exporter/kafkaexporter/README.md

```diff
@@ -88,8 +88,10 @@ The following settings can be optionally configured:
 - `retry_on_failure`
   - `enabled` (default = true)
   - `initial_interval` (default = 5s): Time to wait after the first failure before retrying; ignored if `enabled` is `false`
+  - `randomization_factor` (default = 0.5): Is the random factor used to calculate the next backoffs.
+  - `multiplier` (default = 1.5): Is the value multiplied by the backoff innterval bounds.
   - `max_interval` (default = 30s): Is the upper bound on backoff; ignored if `enabled` is `false`
-  - `max_elapsed_time` (default = 120s): Is the maximum amount of time spent trying to send a batch; ignored if `enabled` is `false`
+  - `max_elapsed_time` (default = 300s): Is the maximum amount of time spent trying to send a batch; ignored if `enabled` is `false`
```

Comment:
[**For reviewers**] I was deliberating on removing the defaults and redirecting to the [exporterhelpers docs](https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/exporterhelper/README.md) but didn't do this yet. Let me know what others think about this, it will trade some of the readability experience for less maintenance but the config seems to be in a fairly stable state so I don't expect changes to the defaults.

Reply:
I would suggest we follow lead of other components, if they tell users to follow the link then lets do that. Otherwise, need to bring it all in.

Reply:
Linking sounds good to me too, as long as we use the same defaults.